### DISCUSSION TITLE
Plugin Development to Synchronize Cadwork 3D and 2D Projects

### DISCUSSION THREAD

**Discussion by lucas-ita** [cwapi3d/cwapi3dpython#200] [Q&A]
Hi,

Would it be possible to use the Cadwork API to create a plugin that allows real-time updates (or not) of a project in Cadwork 3D, which then automatically updates the corresponding project in Cadwork 2D?

**Comment by jspaquet** [comment]
Unfortunately there is no public API available in Cadwork 2D, so you wouldn't be able to automate anything on that side.
